roamance/src/components/social/post/post-creator.tsx

```
@@ -0,0 +1,422 @@
+                        </div>
+                      ))}
+
+                      {videoPaths.map((path, index) => (
+                        <div
+                          key={`vid-${index}`}
+                          className="relative w-24 h-20 rounded-lg overflow-hidden border border-gray-200 dark:border-gray-700 group"
+                        >
+                          <video
+                            src={path}
+                            className="w-full h-full object-cover"
+                          />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)